packages/app/styles/SearchPage.module.scss

```diff
@@ -1,11 +1,37 @@
 @import './variables';
 
+/* stylelint-disable selector-pseudo-class-no-unknown */
 .filtersPanel {
   width: 220px;
   max-height: 100%;
   background-color: #00000030;
   flex-shrink: 0;
   word-break: break-all;
+
+  :global {
```

Comment:
nit: this is probably better to move to a global 'mantine-overrides.scss' file? otherwise there might be wrong styling when SearchPage isn't imported